packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -225,7 +232,8 @@ export const getBuildType = (
  */
 export const prepareDiagnosticMetricItem = (webex: any, item: any) => {
   const origin: Partial<Event['origin']> = {
-    buildType: exports.getBuildType(
```

Comment:
**Question**: Why did this work? We have syntax like `module.exports` in node.js but this syntax seems bit weird and I also don't see any custom object named `exports` being used.

Reply:
It works alright in Modules (NPM Package) because even though there's no explicit mention of `module.exports`, the `export const` syntax means that it is added as part of the exports object.

It was failing in CDN all this while. That is what we are trying to fix

@rarajes2